modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -72,7 +72,7 @@ If the invalid data are entered at any point, the newly created instance is show
 As soon as the authentication data is available on the server, the list of available products is updated.
 
 Available products are all versions of the same product family and architecture as the one installed in the {payg} instance. 
-For example, if the instance has the {sles}{nbsp}15 SP1 product installed, {sles}{nbsp}15 SP2, {sles}{nbsp}15 SP3, {sles}{nbsp}15 SP4 and {sles}{nbsp}15 SP5 are automatically shown in [guimenu]``Admin > Setup Wizard > Products``. 
+For example, if the instance has the {slsa}{nbsp}15 SP1 product installed, {slsa}{nbsp}15 SP2 and {slsa}{nbsp}15 SP3 are automatically shown in [guimenu]``Admin > Setup Wizard > Products``. 
```

Comment:
No, SP4 and SP5 are lost.

That change should be applied in reverse direction in 4.2 branch...